typhos/positioner.py

```diff
@@ -385,6 +410,24 @@ def high_limit_travel_attribute(self):
     def high_limit_travel_attribute(self, value):
         self._high_limit_travel_attr = value
 
+    @QtCore.Property(str, designable=True)
+    def velocity_attribute(self):
+        """The attribute name for the velocity signal."""
+        return self._velocity_attr
+
+    @velocity_attribute.setter
+    def velocity_attribute(self, value):
+        self._velocity_attr = value
+
+    @QtCore.Property(str, designable=True)
+    def acceleration_attribute(self):
+        """The attribute name for the acceleration time signal."""
+        return self._acceleration_attr
+
+    @acceleration_attribute.setter
+    def acceleration_attribute(self, value):
+        self._acceleration_attr = value
+
```

Comment:
Since you are here 😄 ... Should also the limits (switch and travel) be configurable as you did for the velocity and acceleration?

Reply:
Those are already configurable, this snippit is actually just copy/pasted from those and modified.

Reply:
Ooops... missed that...